test/analytics/__fixtures__/plugins/analytics_plugin_a/server/plugin.ts

```diff
@@ -70,7 +70,11 @@ export class AnalyticsPluginAPlugin implements Plugin {
 
         return res.ok({
           body: stats
-            .filter((counter) => counter.event_type === eventType)
+            .filter(
+              (counter) =>
+                counter.event_type === eventType &&
+                ['client', 'FTR-shipper'].includes(counter.source)
```

Comment:
In these tests we are only interested in those stats. If we don't filter, we'll randomly get other shippers reporting, making it more flaky.